app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -44,16 +47,15 @@ trait UserRoutes extends Authentication with PaginationDirectives with UserError
     }
   }
 
-  def listUsers: Route = authenticate { user =>
+  def listUsers: Route = authenticateRootMember { user =>
```

Comment:
Same comment here as with the organizations list, although this might be subject to intra-organization permissions as well (administrator vs regular user).